.github/workflows/go.yml

```diff
@@ -78,6 +78,21 @@ jobs:
     - name: Test registry server with PostgreSQL
       run: go test ./server/registry -postgresql
```

Comment:
Should we be clearing the test cache here?

Reply:
It won't do anything as `go test` hasn't been run before.

Reply:
go test is being run on line 65 and again on line 79. Should we clear the cache before the run on line 79?

Reply:
From the runtime of the test, it appears to be doing something but there are no code changes. I wonder if changing the test flags is enough to clear the cache. *off to read docs*

Reply:
"To disable test caching, use any test flag or argument other than the cacheable flags" https://pkg.go.dev/cmd/go/internal/test

Reply:
JFYI, a quick trick to bypass the cache is to add a count flag, ie. `go test ./... -count=1`.